<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MediaExport_Widget</class>
 <widget class="QWidget" name="MediaExport_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Media Export Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_save_by_frame_name">
       <property name="text">
        <string>Save by Frame Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="save_by_frame_name_checkbox">
       <property name="toolTip">
        <string>If checked, uses the original frame name/ID from media source. If unchecked, uses prefix + padded number.</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_frame_id_padding">
       <property name="text">
        <string>Frame ID Padding:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="frame_id_padding_spinbox">
       <property name="toolTip">
        <string>Number of digits for frame number in filename (e.g., 7 for 0000001.png).</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="value">
        <number>7</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_image_name_prefix">
       <property name="text">
        <string>Image Name Prefix:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="image_name_prefix_edit">
       <property name="toolTip">
        <string>Prefix for image filenames when not saving by original frame name (e.g., &quot;img_&quot;).</string>
       </property>
       <property name="text">
        <string>img</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_image_folder">
       <property name="text">
        <string>Subfolder for Images:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="image_folder_edit">
       <property name="toolTip">
        <string>Name of the subfolder where images will be saved (e.g., &quot;frames&quot;, &quot;images&quot;).</string>
       </property>
       <property name="text">
        <string>images</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
